### PR TITLE
More GitHub actions update

### PR DESCRIPTION
**Pull request by lldelisle**
Last time I forgot 2...

### CODE REVIEW

File: .github/workflows/workflow_test.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ jobs:
       with:
         mode: combine
         html-report: true
-    - uses: actions/upload-artifact@v2.0.1
+    - uses: actions/upload-artifact@v3.0.1
```
Comment (bgruening) [inline .github/workflows/workflow_test.yml:90]: Remove the minor version here as well?

Comment (lldelisle) [reply .github/workflows/workflow_test.yml:90]: Oups

Comment (lldelisle) [reply .github/workflows/workflow_test.yml]: Thanks

**Comment by mvdbeek** [comment]
Thank you @lldelisle!

**Pull request merged**